test/E2ETests/remoteDeploymentConfig.json

```diff
@@ -0,0 +1,9 @@
+﻿{
+  "NanoServer": {
```

Comment:
I was thinking that since this has info about server and credentials, on the CI, we could set the environment variables which could get picked up